www/docs/en/dev/guide/platforms/win8/index.md

```diff
@@ -49,37 +49,23 @@ on how to support IE along with comparable WebKit browsers.
 
 To develop apps for Windows platform you need:
 
-- A Windows 8.1, 32 or 64-bit machine (_Home_, _Pro_, or _Enterprise_ editions) with minimum 4 GB of RAM.
-
-- Windows 8.0, 8.1 or 10, 32 or 64-bit _Home_, _Pro_, or _Enterprise_
-  editions, along with
-  [Visual Studio 2012 Express](http://www.visualstudio.com/downloads) 
-  or Visual Studio 2013.  Visual Studio 2015 is not able to build Windows 8.0 apps.
-
-To develop apps for Windows 8.0 and 8.1 (including Windows Phone 8.1):
-
-- Windows 8.1 or Windows 10, 32 or 64-bit _Home_, _Pro_, or _Enterprise_ editions,
-  along with 
-  [Visual Studio 2013 Express](http://www.visualstudio.com/downloads)
-  or higher. An evaluation version of Windows 8.1 Enterprise is
+- A Windows 8.1, 32 or 64-bit machine (_Home_, _Pro_, or _Enterprise_ editions) 
+  with minimum 4 GB of RAM along with [Visual Studio 2015](http://www.visualstudio.com/downloads) 
```

Comment:
@sarangan12, could you please clarify which link do you mean?
There is a link to the Windows evaluation version below:

>  An evaluation version of Windows 8.1 Enterprise is available from the [Microsoft Developer Network](http://msdn.microsoft.com/en-US/evalcenter/jj554510).